pjlib/src/pj/sock_qos_symbian.cpp

```diff
@@ -21,7 +21,7 @@
 PJ_DEF(pj_status_t) pj_sock_set_qos_params(pj_sock_t sock,
                                            pj_qos_params *param)
 {
-    PJ_ASSERT_RETURN(sock!=0 && sock!=PJ_INVALID_SOCKET, PJ_EINVAL);
+    PJ_ASSERT_RETURN(sock>=0 && sock!=PJ_INVALID_SOCKET, PJ_EINVAL);
```

Comment:
- Better not change anything here in Symbian as it is no longer supported/maintained platform?
- Zero/NULL sock check in Symbian seems to be correct already as `pj_sock_t` is an object (instead of a simple integer file descriptor like in *nix/Win platforms).